src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -346,6 +395,10 @@ private void InitializeSysFS()
 
         private void Initialize()
         {
+            uint gpioRegisterOffset = 0;
```

Comment:
NIT: no need to do this assignment, 0 is the default value for uint.

Reply:
The compiler complains if it is not set somewhere.